src/macaron/slsa_analyzer/checks/provenance_available_check.py

```diff
@@ -93,42 +364,45 @@ def run_check(self, ctx: AnalyzeContext, check_result: CheckResult) -> CheckResu
         CheckResultType
             The result type of the check (e.g. PASSED).
         """
-        ci_services = ctx.dynamic_data["ci_services"]
-        for ci_info in ci_services:
-            ci_service = ci_info["service"]
-            # Checking if a CI service is discovered for this repo.
-            if isinstance(ci_service, NoneCIService):
-                continue
-            # Only get the latest release.
-            release = ci_service.api_client.get_latest_release(ctx.component.repository.full_name)
-            if release:
-                # Store the release data for other checks.
-                ci_info["latest_release"] = release
-
-                # Get the provenance assets.
-                for prov_ext in defaults.get_list("slsa.verifier", "provenance_extensions"):
-                    assets = ci_service.api_client.get_assets(release, ext=prov_ext)
-                    if not assets:
-                        continue
+        provenance_extensions = defaults.get_list(
+            "slsa.verifier",
+            "provenance_extensions",
+            fallback=["intoto.jsonl"],
+        )
 
-                    # Store the provenance assets for other checks.
-                    ci_info["provenance_assets"].extend(assets)
+        # We look for the provenances in the package registries first, then CI services.
+        # (Note the short-circuit evaluation with OR.)
+        try:
+            provenance_assets = self.find_provenance_assets_on_package_registries(
+                repo_full_name=ctx.component.repository.full_name,
```

Comment:
I think you need to pass the file system path to the clone directory here?
```suggestion
                repo_full_name=ctx.component.repository.fs_path,
```

Reply:
Fixed in 8a9f82be1bf53574798a0a9f3de8ff9ac843c484.